src/CairoMakie.jl

```diff
@@ -49,34 +52,48 @@ struct CairoScreen{S} <: AbstractPlotting.AbstractScreen
     scene::Scene
     surface::S
     context::CairoContext
-    pane::Nothing#Union{CairoGtkPane, Void}
+    pane::Nothing # TODO: Union{CairoGtkPane, Void}
 end
-# # we render the scene directly, since we have no screen dependant state like in e.g. opengl
+
+# we render the scene directly, since we have no screen dependant state like in e.g. opengl
 Base.insert!(screen::CairoScreen, scene::Scene, plot) = nothing
 
 function Base.show(io::IO, ::MIME"text/plain", screen::CairoScreen{S}) where S
     println(io, "CairoScreen{$S} with surface:")
     println(io, screen.surface)
 end
 
-# Default to Window+Canvas as backing device
-function CairoScreen(scene::Scene)
-    w, h = size(scene)
-    surf = CairoARGBSurface(w, h)
+# Default to ARGB Surface as backing device
+# TODO: integrate Gtk into this, so we can have an interactive display
+function CairoScreen(scene::Scene; device_scaling_factor = 1, antialias = Cairo.ANTIALIAS_BEST)
+    w, h = round.(Int, scene.camera.resolution[] .* device_scaling_factor)
+    surf = Cairo.CairoARGBSurface(w, h)
+
+    # this sets a scaling factor on the lowest level that is "hidden" so its even
+    # enabled when the drawing space is reset for strokes
+    # that means it can be used to increase or decrease the image resolution
+    ccall((:cairo_surface_set_device_scale, Cairo.libcairo), Cvoid, (Ptr{Nothing}, Cdouble, Cdouble),
+        surf.ptr, device_scaling_factor, device_scaling_factor)
+
     ctx = CairoContext(surf)
-    CairoScreen(scene, surf, ctx, nothing)
+    Cairo.set_antialias(ctx, antialias)
+
+    return CairoScreen(scene, surf, ctx, nothing)
 end
 
-function CairoScreen(scene::Scene, path::Union{String, IO}; mode = :svg, device_scaling_factor = 1)
+function CairoScreen(scene::Scene, path::Union{String, IO}; mode = :svg, device_scaling_factor = 1, antialias = Cairo.ANTIALIAS_BEST)
```

Comment:
i would just remove the `mode = :svg` default and make it a required keyword. it's not obvious why svg would be a meaningful default and then you don't know what's happening in code that calls this